src/main/java/com/xiaomi/infra/pegasus/client/PegasusTable.java

```diff
@@ -1067,17 +1067,62 @@ public MultiGetResult multiGet(
       int timeout /*ms*/)
       throws PException {
     if (timeout <= 0) timeout = defaultTimeout;
+
+    String hashKeyStr = hashKey == null ? "" : new String(hashKey);
+    String startSortKeyStr = startSortKey == null ? "" : new String(startSortKey);
+    String stopSortKeyStr = stopSortKey == null ? "" : new String(stopSortKey);
```

Comment:
Not all hashkey/sortkey/value use UTF-8 encoding. Are you aware of this?
Printing String from bytes may display unprintable characters, which may not be confusing rather than helpful to users.

I recommend we should simply offer the problem to the caller.

```
new TimeoutException("getting scanner times out")
```